runhouse/utils.py

```diff
@@ -800,3 +810,335 @@ def print_sky_clusters_msg(num_sky_clusters: int):
         console.print(
             f"{msg} For more information, please run [bold italic]sky status -r[/bold italic]."
         )
+
+
+def adjust_resource_type(resource_type: str):
```

Comment:
maybe `condense_resource_type` is more accurate? meaning we're abbreviating the full resource path for display purposes right?